pilot/pkg/security/authz/builder/builder_test.go

```diff
@@ -113,13 +114,13 @@ func TestBuilder_BuildHTTPFilter(t *testing.T) {
 				policy.SimpleBinding("binding-1", "a", "role-1"),
 				policy.SimpleAuthzPolicy("authz-bar", "a"),
 			},
-			wantPolicies: []string{"role-1", "authz-v1beta1-merged[authz-bar]"},
+			wantPolicies: []string{"authz-bar[0]"},
```

Comment:
Just to make sure I understand correctly, does this mean the expected output is authz only, the legacy policies are ignored?

Reply:
yes. if both new and old policies are specified for a workload, only the new policy will be used.